gap/examples.gi

```diff
@@ -370,3 +370,33 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallGlobalFunction(DIGRAPHS_KingsGraph,
+function(n, k)
+  local D, i, j;
+  if (n < 1) or (k < 1) then
```

Comment:
This could return an error if n or k is incomparable with 1, ie a function or digraph so a check should be put in place for n and k being integers before this check.